dbt/include/snowflake/macros/materializations/dynamic_table/materialization.sql

```diff
@@ -93,27 +92,31 @@
 {% endmacro %}
 
 
-{% macro dynamic_table_execute_no_op(target_relation) %}
+{% macro dynamic_table_execute_no_op(dynamic_table) %}
     {% do store_raw_result(
         name="main",
-        message="skip " ~ target_relation,
+        message="skip " ~ dynamic_table.name,
         code="skip",
         rows_affected="-1"
     ) %}
 {% endmacro %}
 
 
-{% macro dynamic_table_execute_build_sql(build_sql, existing_relation, target_relation, post_hooks) %}
+{% macro dynamic_table_execute_build_sql(build_sql, dynamic_table, existing_relation, post_hooks) %}
 
     {% set grant_config = config.get('grants') %}
 
+    {% set original_query_tag = set_query_tag() %}
```

Comment:
Where did this come from/why? I don't see anything in your issue spec describing it!

Reply:
Every other materialization in `dbt-snowflake` did this. And it seemed like a functionality we supported across the adapter; so I added it in.

Reply:
Ah makes sense! Good catch.